modules/core/03-connection/keeper/keeper.go

```diff
@@ -208,6 +208,25 @@ func (k Keeper) GetAllConnections(ctx sdk.Context) (connections []types.Identifi
 	return connections
 }
 
+// GetSentinelLocalhostConnectionEnd returns the sentinel localhost connection end.
+func (k Keeper) GetSentinelLocalhostConnectionEnd() types.ConnectionEnd {
+	counterparty := types.NewCounterparty(exported.Localhost, types.LocalhostID, commitmenttypes.NewMerklePrefix(k.GetCommitmentPrefix().Bytes()))
+	return types.ConnectionEnd{
+		State:        types.OPEN,
+		ClientId:     exported.Localhost,
+		Versions:     types.ExportedVersionsToProto(types.GetCompatibleVersions()),
+		Counterparty: counterparty,
+	}
+}
+
+// CreateLocalhostConnectionEnd writes a sentinel localhost ConnectionEnd to state.
+func (k Keeper) CreateLocalhostConnectionEnd(ctx sdk.Context) {
```

Comment:
Do we really need this function? Cannot we use the existing `SetConnection` and pass as argument the sentinel localhost connection end created by `GetSentinelLocalhostConnectionEnd`?